ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands2.java

```diff
@@ -767,7 +709,7 @@ public void testNonAcidToAcidConversion3() throws Exception {
     // 4. Update the existing row, and insert another row to newly-converted ACID table
     runStatementOnDriver("update " + Table.NONACIDORCTBL + " set b=3 where a=1");
     runStatementOnDriver("insert into " + Table.NONACIDORCTBL + "(a,b) values(3,4)");
-    status = fs.listStatus(new Path(TEST_WAREHOUSE_DIR + "/" +
+    status = fs.listStatus(new Path(getWarehouseDir() + "/" +
```

Comment:
Question: Would it worth to create a method for this?
`listStatus(String tableName)`

Reply:
I would keep the existing signature for now, it's more generic, allows to work with partitioned tables + provide diff filters.